test/whl-scale-tests/geneva-config-files/text_log_geneva_config.xml

```diff
@@ -0,0 +1,82 @@
+      AgentResourceUsage element's attributes.
+    -->
+    <AgentResourceUsage cpuPercentUsage="25" cpuThrottling="true" memoryLimitInMB="1000" memoryThrottling="true" diskQuotaInMB="10000" />
+    <AgentMetrics mdmMonitoringAccount="GENEVA_METRIC_ACCOUNT" />
+  </Management>
+  <!-- Events documentation: https://jarvis.dc.ad.msft.net/?page=documents&section=9c95f4eb-8689-4c9f-81bf-82d688e860fd&id=ac0084ad-5065-4b16-8f7d-0a5193143378   -->
+  <!--
+      Events can contain one or more Event elements.  The Event elements each describe a different kind of collector.
+      Events must be defined in the order of the mds2.xsd.
+    -->
+  <Events>
+    <CounterSets storeType="Local" mdmMonitoringAccount="GENEVA_METRIC_ACCOUNT" format="Factored">
+      <CounterSet eventName="CounterTable" sampleRateInSeconds="60" mdmNamespace="PlatformMetrics">
+        <MdmDimensions>
+          <MdmDimension name="CloudTenant"><![CDATA[GetEnvironmentVariable("AKS_CLUSTER_NAME")]]></MdmDimension>
+          <MdmDimension name="CloudRole"><![CDATA[GetEnvironmentVariable("HOSTNAME")]]></MdmDimension>
+          <MdmDimension name="RoleInstance"><![CDATA[GetEnvironmentVariable("AKS_REGION")]]></MdmDimension>
+          <MdmDimension name="Environment"><![CDATA[GetEnvironmentVariable("MONITORING_ENVIRONMENT")]]></MdmDimension>
+          <MdmDimension name="Datacenter"><![CDATA[GetEnvironmentVariable("MONITORING_DATACENTER")]]></MdmDimension>
+          <MdmDimension name="ScaleUnit"><![CDATA[GetEnvironmentVariable("MONITORING_TENANT_NAME")]]></MdmDimension>
+        </MdmDimensions>
+        <!-- System counters -->
+        <Counter>\Memory\Available MBytes</Counter>
+        <Counter>\System\System Calls/sec</Counter>
+        <Counter>\Processor(_Total)\% Processor Time</Counter>
+        <Counter>\Process(_Total)\Working Set</Counter>
+        <Counter>\LogicalDisk(*)\% Free Space</Counter>
+        <Counter>\LogicalDisk(*)\% Disk Read Time</Counter>
+        <Counter>\LogicalDisk(*)\% Disk Write Time</Counter>
+      </CounterSet>
+      <CounterSet eventName="IisCounters" sampleRateInSeconds="60" mdmNamespace="PlatformMetrics">
+        <MdmDimensions>
+          <MdmDimension name="CloudTenant"><![CDATA[GetEnvironmentVariable("AKS_CLUSTER_NAME")]]></MdmDimension>
+          <MdmDimension name="CloudRole"><![CDATA[GetEnvironmentVariable("HOSTNAME")]]></MdmDimension>
+          <MdmDimension name="RoleInstance"><![CDATA[GetEnvironmentVariable("AKS_REGION")]]></MdmDimension>
+          <MdmDimension name="Environment"><![CDATA[GetEnvironmentVariable("MONITORING_ENVIRONMENT")]]></MdmDimension>
+          <MdmDimension name="Datacenter"><![CDATA[GetEnvironmentVariable("MONITORING_DATACENTER")]]></MdmDimension>
+          <MdmDimension name="ScaleUnit"><![CDATA[GetEnvironmentVariable("MONITORING_TENANT_NAME")]]></MdmDimension>
+        </MdmDimensions>
+      </CounterSet>
+    </CounterSets>
+    <TextLogSubscriptions>
```

Comment:
We should make a note here that this won't use the Managed Fluent feature

Reply:
Added example xml for managed fluent here. Also documented how to switch to managed fluent in the md

Reply:
Looks good to me